src/functions.c

```diff
@@ -561,45 +695,45 @@ void functionFlushCommand(client *c) {
 
 void functionHelpCommand(client *c) {
     const char *help[] = {
-"CREATE <ENGINE NAME> <FUNCTION NAME> [REPLACE] [DESC <FUNCTION DESCRIPTION>] <FUNCTION CODE>",
-"    Create a new function with the given function name and code.",
-"DELETE <FUNCTION NAME>",
-"    Delete the given function.",
-"INFO <FUNCTION NAME> [WITHCODE]",
-"    For each function, print the following information about the function:",
-"    * Function name",
-"    * The engine used to run the function",
-"    * Function description",
-"    * Function code (only if WITHCODE is given)",
+"LOAD <ENGINE NAME> <LIBRARY NAME> [REPLACE] [DESC <LIBRARY DESCRIPTION>] <LIBRARY CODE>",
+"    Create a new library with the given library name and code.",
+"DELETE <LIBRARY NAME>",
+"    Delete the given library.",
+"INFO <LIBRARY NAME> [WITHCODE]",
+"    Print the following information about the library:",
+"    * Library name",
+"    * The engine used to run the library",
+"    * Library description",
+"    * Library code (only if WITHCODE is given)",
 "LIST",
-"    Return general information on all the functions:",
-"    * Function name",
-"    * The engine used to run the function",
-"    * Function description",
+"    Return general information on all the libraries:",
+"    * Library name",
+"    * The engine used to run the Library",
+"    * Library description",
 "STATS",
 "    Return information about the current function running:",
 "    * Function name",
 "    * Command used to run the function",
 "    * Duration in MS that the function is running",
-"    If not function is running, return nil",
+"    If function is not running, return nil",
```

Comment:
i think it should be
```suggestion
"    If no function is running, return nil",
```